MdeModulePkg/Application/UiApp/FrontPageCustomizedUiSupport.c

```diff
@@ -191,6 +191,11 @@ UiSupportLibCallbackHandler (
 
   if (Action == EFI_BROWSER_ACTION_RETRIEVE) {
     if (QuestionId == FRONT_PAGE_KEY_LANGUAGE) {
+      if (Value == NULL) {
+        *Status = EFI_INVALID_PARAMETER;
+        return TRUE;

```

Comment:
Should this return FALSE? Per the function header, TRUE indicates the callback successfully handled the action, FALSE indicates that the callback is not supported by this handler.

Reply:
I generally agree and will check the surrounding logic to confirm.